app/scripts/services/builds.js

```diff
@@ -108,16 +109,18 @@ angular.module("openshiftConsole")
     var cloneBuild = function(originalBuild, buildConfigName) {
       var buildType = isJenkinsPipelineStrategy(originalBuild) ? 'pipeline' : 'build';
       var originalDisplayName = getBuildDisplayName(originalBuild, buildConfigName);
+
       var req = {
         kind: "BuildRequest",
-        apiVersion: "v1",
+        apiVersion: originalBuild.apiVersion,
```

Comment:
This should be the api version for `buildsCloneVersion`, which might not match `originalBuild.apiVersion`

Reply:
Ah, so `APIService.toAPIVersion(buildsCloneVersion)`, ill update.

Reply:
I bet I'm gonna be doing a # of bug fixes around this.  Now it's getting fiddly.

Reply:
updated